reactive/calico.py

```diff
@@ -562,6 +567,8 @@ def ready():
         'calico.cni.configured', 'calico.bgp.globals.configured',
         'calico.node.configured', 'calico.bgp.peers.configured'
     ]
+    if is_state('upgrade.series.in-progress'):
+        return
```

Comment:
Maybe make this

```
if is_state('upgrade.series.in-progress'):
    status_set('blocked', 'Series upgrade in progress')
    return
```

to ensure that the blocked status does not get clobbered by handlers.